tests/conftest.py

```diff
@@ -127,14 +127,27 @@ def homedir(i18n):
 
 
 @pytest.fixture(scope="function")
-def functional_test_app_started_context(homedir, reply_status_codes, session, config, qtbot):
+def export_service():
+    """An export service that assumes the Qubes RPC calls are successful and skips them."""
+    export_service = export.Service()
+    # Ensure the export_service doesn't rely on Qubes OS:
+    export_service._run_disk_test = lambda dir: print("_run_disk_test")
+    export_service._run_usb_test = lambda dir: print("_run_usb_test")
+    export_service._run_disk_export = lambda dir, path, pasphrase: print("_run_disk_export")
+    return export_service
+
+
+@pytest.fixture(scope="function")
+def functional_test_app_started_context(
+    homedir, reply_status_codes, session, config, qtbot, export_service
+):
     """
     Returns a tuple containing the gui window and controller of a configured client. This should be
     used to for tests that need to start from the login dialog before the main application window
     is visible.
     """
     app_state = state.State()
-    gui = Window(app_state)
+    gui = Window(app_state, export_service)
```

Comment:
The existing functional tests require a modified `export_service` that doesn't trigger Qubes OS-spcific commands. A fixture provides that in this case.

Reply:
Makes sense, and thank you for the explanations throughout :)